Enyim.Caching/Configuration/MemcachedClientConfiguration.cs

```diff
@@ -154,9 +154,6 @@ public MemcachedClientConfiguration(
             {
                 try
                 {
-                    if (options.Transcoder == "BinaryFormatterTranscoder")
```

Comment:
Will will have to remove the setter for `BinaryFormatterTranscoder` as well

Reply:
This comes from Appsettings